src/converter.js

```diff
@@ -16,14 +16,16 @@ function conversions(comment) {
   const formattedConversions = ch.formatConversion(roundedConversions);
   
   return formattedConversions.reduce((memo, conversion) => {
-    const key = conversion['imperial']['numbers'].join('-') + conversion['imperial']['unit'];
+    let joiner = "";
+    if("joiner" in conversion['imperial']) joiner = " " + conversion['imperial']['joiner'] + " ";
```

Comment:
I know we don't have a style guide yet, but can you turn this if statement to multi-line with curly brackets? Single line if statements confuse me :(